uber/config.py

```diff
@@ -195,6 +195,11 @@ def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
     @property
```

Comment:
Oh right, of course. Oddly enough this did actually work during testing, but I'll go ahead and return the dict instead.